scripts/v2/generate-helm-manifest.sh

```diff
@@ -49,6 +49,10 @@ sed -i "s@$LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE@{{.Values.image.repository}}@g
 # Perl multiline replacements - using this because it's tricky to do these sorts of multiline replacements with sed
 perl -0777 -i -pe 's/(template:\n.*metadata:\n.*annotations:\n(\s*))/$1\{\{- if .Values.podAnnotations \}\}\n$2\{\{ toYaml .Values.podAnnotations \}\}\n$2\{\{- end \}\}\n$2/igs' "$GEN_FILES_DIR"/*_deployment_* # Add pod annotations
 
+# Affinity and Toleration
+sed -i '/    spec:/a\    {{- with .Values.affinity }}\n      affinity:\n        {{- toYaml . | nindent 8 }}\n    {{- end }}' "$GEN_FILES_DIR"/*_deployment_*
```

Comment:
You might wanna steal the (horrible) perl line above as it captures spaces and so deals with variable indentation?

I assume that you've tested and this works have we have it today... we really should probably fix how we generate the Helm template (might be better to just duplicate stuff, I dunno)